assets/css/sensei-course-theme/notices.scss

```diff
@@ -1,22 +1,24 @@
+@import '~@wordpress/base-styles/breakpoints';
 @import '../notices';
-@import './quiz-graded';
-
-
-$mobile_break_point: 782px;
 
 body.sensei-course-theme {
 	& > .sensei-message {
 		display: none;
 	}
 }
 
+/* All notices */
 .sensei-course-theme .sensei-lms-notice {
-	background-color: rgba(125, 125, 125, 0.08);
-	border-color: var(--border-color);
-	color: inherit;
-	padding: 1.125rem 1.5rem;
+	background-color: var(--wp--preset--color--primary, #f6f7f7);
```

Comment:
One thing I noticed while reviewing the other PR based on this one is that maybe this background could be a little weird depending on the theme colors.

Here is what happens with the TT3:

<img width="924" alt="Screenshot 2023-10-20 at 15 36 20" src="https://github.com/Automattic/sensei/assets/876340/818a280d-2fe7-4f54-9ece-6e35b69dc7e1">

I tried to think of a suggestion here, but I couldn't think of any suggestion that wouldn't be to have a default color not dependent on the theme. And maybe override it specifically in the Course theme with the color we want?

Reply:
@renatho Ohhhh...interesting! In https://github.com/Automattic/sensei/pull/7197/commits/196fd9644a3afb1648e984bf3621ded20566bd24, I ended up switching to use the foreground color. For the Course theme, primary is the same color as foreground. Now, if a block theme sets both the foreground and background colors, they should work well together. There may still be a small number of cases where it doesn't work for some themes, but as long as it's not a top Sensei theme, I think it's fine.

Reply:
Thank you @donnapep! And the color looks good to me on TT3 now. :)